Modules/Scaler/CommonScaler.py

```diff
@@ -990,9 +984,9 @@ def _estimate_resolution_limit(
         m.set_limit_cc_half(params.cc_half)
         m.set_cc_half_fit(params.cc_half_fit)
         m.set_cc_half_significance_level(params.cc_half_significance_level)
-        if use_isigma:
+        if params.isigma:
```

Comment:
There is no need for this if statement here as setting the limit isn't what determines what is actually used for the cutoff.